test/executor/copy_test.cpp

```diff
@@ -43,18 +43,20 @@ TEST_F(CopyTests, Copying) {
   catalog->CreateDatabase("emp_db", txn);
   txn_manager.CommitTransaction(txn);
 
-  optimizer::Optimizer optimizer;
+  std::unique_ptr<optimizer::AbstractOptimizer> optimizer;
```

Comment:
Switch this initialization to be one line:

`std::unique_ptr<optimizer::AbstractOptimizer> optimizer(new optimizer::Optimizer);`